platforms/iOS/vm/OSX/sqSqueakOSXApplication+events.m

```diff
@@ -73,6 +73,8 @@ @implementation sqSqueakOSXApplication (events)
 // If the event does not correspond to this window, we take it from the event queue anyways and re-post it afterwards
 // This gives other windows the opportunity to consume their events
 - (void) pumpRunLoopEventSendAndSignal:(BOOL)signal {
+
+	@autoreleasepool {
        NSEvent *event;
        NSMutableArray *alienEventQueue = [[NSMutableArray alloc] init];
```

Comment:
@johnmci tries to make sure that everything builds on ARC and non-ARC, so maybe do this:
```suggestion
       NSMutableArray *alienEventQueue = AUTORELEASEOBJ([[NSMutableArray alloc] init]);
```